src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs

```diff
@@ -104,6 +104,20 @@ private void AddProject(SlnFile slnFile, string projectPath)
                 };
 
                 slnFile.Projects.Add(slnProject);
+
+                if (slnFile.ProjectConfigurationsSection.Any())
+                {
+                    var existingBuildConfigs = slnFile.ProjectConfigurationsSection.First();
```

Comment:
How about then maybe creating an extension method or a private method here called GetFirstValidProjectConfigurationSection? That would make this part of the code clearer, I think.